src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,19 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[3];
+
+        machines[0] = new Truck();
+        machines[1] = new Bulldozer();
+        machines[2] = new Excavator();
```

Comment:
It's better to use shorter forms when there is no impact to readability
```suggestion
        Machine[] machines = { new Truck(), new Bulldozer(), new Excavator() };
```